Fitness-v2-server/internal/controllers/notification_controller.go

```diff
@@ -122,9 +155,31 @@ func MarkNotificationAsRead(c echo.Context) error {
 		return c.JSON(http.StatusOK, map[string]string{
 			"message": "Notification marked as read",
 		})
+
+	case models.NotificationTypeUserScoreApproved:
+	case models.NotificationTypeUserScoreRejected:
+	case models.NotificationTypeUserScorePending:
+		err := config.Queries.MarkNotificationAsReadByNotificationID(c.Request().Context(), markNotificationAsReadReq.ID)
+		if err != nil {
+			utils.FmtLogError(
+				"notification_controller.go",
+				"MarkNotificationAsRead",
+				fmt.Errorf("failed to mark notification as read: %s", err),
+			)
+			return echo.NewHTTPError(http.StatusInternalServerError, map[string]string{
+				"message": "Failed to mark notification as read, trouble with server",
+			})
+		}
```

Comment:
@coderabbitai because I bind the id to a UUID in the begining of this function, how would it be best to check if that uuid exists, as it's an omitempty should I check if markNotificationAsReadReq.ID == nil ? or would another check be better?

Reply:
Implemented